packages/firestore/src/core/firestore_client.ts

```diff
@@ -543,7 +543,9 @@ export class FirestoreClient {
 
   getNamedQuery(queryName: string): Promise<NamedQuery | undefined> {
     this.verifyNotTerminated();
-    return getNamedQuery(this.localStore, queryName);
+    return this.asyncQueue.enqueue(() =>
```

Comment:
Small comment on `FirestoreClient.loadBundle` above. Instead of logging the exception in FirestoreClient, could we log it in `loadBundle()`?. That would remove the catch statement, and also bubble up an UnhandledPromiseRejection for the user if they don't handle the failure.

Reply:
I don't see this change. Can you push it?

Reply:
As discussed, the log is moved into `sync_engine.ts`